contrib/bash/goshy

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -x
+show_help() {
+  cat <<EOF
+Usage: goshy [Options] FILE
+Options:
+  -b --burn              Burn after reading.
+  -h --help              Print this help message.
+  -u --only-url          Print only URL as response.
+  -p --period TIMEFRAME  Set a custom expiry period.
+EOF
+}
+
+while getopts p:period: flag
+do
+  case "$\{flag\}" in
+    p) PERIOD_IS_SET=1;;
+    period) PERIOD_IS_SET=1;;
+  esac
+done > /dev/null 2>&1
+
+if [ -z $PERIOD_IS_SET ]; then
+  PERIOD='3d'
+fi
```

Comment:
This whole block from the `while getopts` loop until the final `if` seems just to over complicate things. Why just don't set `PERIOD='3d'` always, as, if neither `-p` nor `--period` were given by the user, it will get overwritten anyways.
Or am I overseeing something?

 ```suggestion
PERIOD='3d'
```

Reply:
Resolved with 8cb43602bbe9cd1300b9855adab62a342319b308